include/dxc/Support/dxcapi.extval.h

```diff
@@ -1,8 +1,160 @@
 #include "dxc/Support/dxcapi.use.h"
+#include "dxc/WinAdapter.h"
+#include "llvm/Support/raw_ostream.h"
+
 #include <cassert>
 #include <string>
 
 namespace dxc {
+
+class ExternalValidationHelper3 : public IDxcCompiler3 {
```

Comment:
Do these classes actually need to be in the header file?  I don't think they're used by anything other than the CPP file.